src/components/v5/common/ActionSidebar/types.ts

```diff
@@ -13,3 +16,20 @@ export type ErrorBannerProps = {
   title: MessageDescriptor | string;
   actionText?: MessageDescriptor | string;
 };
+
+//
+export type ActionFormOptions = UseFormProps & {
```

Comment:
Can you use interface here? Also, what is `//` in the line above for?

Reply:
Done